### PR TITLE
Issue #292: Update Drone configuration.

### LINKED ISSUE #292: Update Drone configuration.
In order to prevent build issue with the image, we must make sure that the docker images are pulled at each build.

### PR DESCRIPTION
**Pull request by drupol**
Fix #292

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# ==============================================================================
```
Comment (aritomelo) [inline .drone.yml:1]: Why not use the universal way of using PULL, as here and use DRY (don't repeat yourself!):

clone:
  git:
    image: plugins/git:next
    pull: true

Comment (drupol) [reply .drone.yml:1]: Ah, I didn't know!
Then yes, let's use it

Comment (drupol) [reply .drone.yml:1]: I'm wondering if using `pull: true` in there has the same meaning as putting it in the other services.

Comment (aritomelo) [reply .drone.yml:1]: I have not tried it yet, let's see what drone says!

Comment (drupol) [reply .drone.yml:1]: Fixed.

File: .drone.yml [hunk 2]
```diff
@@ -7,6 +18,7 @@ services:
     image: fpfis/httpd-php-dev:${PHP_VERSION=5.6}
     environment:
     - DOCUMENT_ROOT=/test/atomium
+    pull: true
```
Comment (aritomelo) [inline .drone.yml:21]: I think you forget this one, isn't it?

**Review by aritomelo — COMMENTED** [review]
I think you forget this one, isn't it?

File: .drone.yml (hunk 2 above)
Comment (drupol) [reply .drone.yml:21]: Nope, @voidtek told me that the `pull: true` must be in the `services`, not in the `pipeline`.

**Review by aritomelo — APPROVED** [review]
The same here, I am Approving it, however, it still to be clarified whether or not this is really needed once the EC environment already includes this plugin.

**Pull request merged**